src/lib/x509/pkix_enums.h

```diff
@@ -67,6 +67,8 @@ enum class Certificate_Status_Code {
    CERT_CHAIN_TOO_LONG = 4002,
    CA_CERT_NOT_FOR_CERT_ISSUER = 4003,
    NAME_CONSTRAINT_ERROR = 4004,
+   IPADDR_BLOCK_ERROR = 4011,
+   ASBLOCKS_ERROR = 4012,
```

Comment:
We weren't really sure what to call the errors, nor what kind of error messages to use. We think it's definitely a validation error, but unfortunately there wasn't any space left within the enum, so now the numbers are kind of awkward.

Reply:
On the naming, what you have seems fine. It's a bit obscure but anyone doing RPKI probably has at least some idea what these mean.

Re the numbering - oof. Absolutely my bad here. Can you add here // TODO(Botan4) annotations to renumber the enum? I do not know what I was thinking leaving so little (and/or zero) gaps here.